cmds/core/more/more.go

```diff
@@ -23,43 +23,50 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"io"
 	"log"
 	"os"
 
 	flag "github.com/spf13/pflag"
 )
 
 var lines = flag.Int("lines", 40, "screen size in number of lines")
+var errOnlyOneFile = fmt.Errorf("more can only take one file")
+var errLinesMustBePositive = fmt.Errorf("lines must be positive")
 
-func main() {
-	flag.Parse()
-	if flag.NArg() != 1 {
-		log.Fatal("more can only take one file")
+func run(stdin io.Reader, stdout io.Writer, lines int, args []string) error {
```

Comment:
I think in future we'll accept more than one file? I wonder if we might have 
runOneFile(stdin io.Reader, stdout io.Writer, lines int, name string) error {
}
and 
run(stdin io.Reader, stdout io.Writer, lines int, args []string) error {
	if len(args) != 1 {
		return errOnlyOneFile
	}
      return runOneFile(in, out, lines, args[0])
}

Reply:
In this case I think we can allow command to work with arrays of files, and it will simplify the code as well.